src/test/java/org/apache/commons/text/ExtendedMessageFormatTest.java

```diff
@@ -55,11 +55,11 @@ public void testExtendedFormats() {
         final String pattern = "Lower: {0,lower} Upper: {1,upper}";
         final ExtendedMessageFormat emf = new ExtendedMessageFormat(pattern, registry);
         assertEquals("TOPATTERN", pattern, emf.toPattern());
-        assertEquals("Lower: foo Upper: BAR", emf.format(new Object[] {"foo", "bar"}));
-        assertEquals("Lower: foo Upper: BAR", emf.format(new Object[] {"Foo", "Bar"}));
-        assertEquals("Lower: foo Upper: BAR", emf.format(new Object[] {"FOO", "BAR"}));
-        assertEquals("Lower: foo Upper: BAR", emf.format(new Object[] {"FOO", "bar"}));
-        assertEquals("Lower: foo Upper: BAR", emf.format(new Object[] {"foo", "BAR"}));
+        assertEquals("Lower: foo Upper: BAR", emf.format(new Object[]{"foo", "bar"}));
+        assertEquals("Lower: foo Upper: BAR", emf.format(new Object[]{"Foo", "Bar"}));
+        assertEquals("Lower: foo Upper: BAR", emf.format(new Object[]{"FOO", "BAR"}));
+        assertEquals("Lower: foo Upper: BAR", emf.format(new Object[]{"FOO", "bar"}));
+        assertEquals("Lower: foo Upper: BAR", emf.format(new Object[]{"foo", "BAR"}));
```

Comment:
Unless whitespace changes like this one are required by Checkstyle, please consider reverting them to prevent polluting the Git history.